addons/website/views/snippets/snippets.xml

```diff
@@ -985,7 +985,7 @@
 
     <!-- Stretch section -->
     <div data-js="ContainerWidth" data-selector="section, .s_carousel .carousel-item, s_quotes_carousel .carousel-item"
-        data-target="> .container, > .container-fluid, > .o_container_small">
+        data-target="> .container, > .container-fluid, > .o_container_small" data-exclude="#o_wblog_post_content *">
```

Comment:
I am not sure what this is fixing? Seems in conflict with other task, can you explain/link a screenshot?

Reply:
It was linked to that specification :

User is not able to change the witdh of the first content of a blog because it's not part of a blog. It's weird for the user that 'content width' buttno have no effect here. https://www.awesomescreenshot.com/video/2549246?key=685df43eccbb8d981623c6b95253805d (just exclude the content width option in blog post -> better will be done in master)

The commit message is confusing, it is in fact the option "container width" of every block dragged into a blog that will be ineffective, thus it is hidden

Before :
![image](https://user-images.githubusercontent.com/58561718/110359240-8bb22380-803d-11eb-9663-227075cc50fa.png)

After :
![image](https://user-images.githubusercontent.com/58561718/110359307-a2587a80-803d-11eb-8744-df6a823dc885.png)

Reply:
Mmh, indeed. I was confused by another spec for master. Dragging that kind of block in blogs is already not the intented (because ugly) behavior so it feels kinda pointless to "fix" this problem... anyways we can do it if it is in the specs. However: you are fixing a website_blog behavior inside the website app: you should fix it in website_blog (`o_wblog_post_content` is not an ID of a DOM node the website app is supposed to know about). So an xpath in website_blog to add this `data-exclude`.

Reply:
Ok, thank you for the explanations, should I move it to this PR for master https://github.com/odoo/odoo/pull/66132 or keep it on this one ?

Reply:
This one :+1: